Uno.Gallery/Uno.Gallery.UITest/Given_AutoSuggestBox.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.UI.Xaml;
+using NUnit.Framework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+
+namespace Uno.Gallery.UITests
+{
+	public class Given_AutoSuggestBox : TestBase
+	{
+		[Test]
+		public void AutoSuggestBox_Default()
+		{
+			NavigateToSample("AutoSuggestBox", "Fluent");
+			//TakeScreenshot("Before Text");
+			var AutoSuggest = new QueryEx(x => x.All().Marked("fluent_AutoSuggestBox"));
+			
+			AutoSuggest.EnterText("PasswordBox");
+			bool isSuggestionListOpen = AutoSuggest.GetDependencyPropertyValue<bool>("IsSuggestionListOpen");
+			Assert.IsTrue(isSuggestionListOpen, "Suggestion list is open");
+			Assert.AreEqual("PasswordBox", AutoSuggest.GetDependencyPropertyValue<String>("Text"));
+
+			AutoSuggest.ClearText();
+			AutoSuggest.EnterText("Button");
+			Assert.IsTrue(isSuggestionListOpen, "Suggestion list is open");
+			Assert.AreEqual("Button", AutoSuggest.GetDependencyPropertyValue<String>("Text"));
+
+			AutoSuggest.ClearText();
+			AutoSuggest.EnterText("RadioButton");
+			Assert.IsTrue(isSuggestionListOpen, "Suggestion list is open");
+			Assert.AreEqual("RadioButton", AutoSuggest.GetDependencyPropertyValue<String>("Text"));
+
+			AutoSuggest.ClearText();
+			Assert.AreEqual("", AutoSuggest.GetDependencyPropertyValue<String>("Text"));
+			Assert.IsTrue(isSuggestionListOpen, "Suggestion list is open");
```

Comment:
You need to call `AutoSuggest.GetDependencyPropertyValue<String>("IsSuggestionListOpen")` again here.

The `isSuggestionListOpen` variable that you created won't automatically update when new text is entered into the autosuggestbox

In this case the suggestion list should not be open when clearing the text so you should be calling `Assert.IsFalse`, not `Assert.IsTrue`